### ISSUE TITLE
Overhaul Experiment Setup

### ISSUE THREAD

**Issue by lokijuhy** [CMI-UZH/auto-discern#38]
**1. Hyperparameter tuning and Validation process:**
 - For each of 5 80/20 train/test splits using StratifiedKFold sampling on the label:
   - Hyperparameter search:
      - Make 90/10 training validation split
      - Run grid search to find best hyperparameters
    - Train best hyperparameter-model on full 80, 
    - return score on 20

**2. Report F1 score**

**3. Report accuracy by topic (arthiritis, breast cancer, depression)**

**4. Compare Random Forest and XGBoost**
